### ISSUE TITLE
Pre-work Checklist: Developer: Sarah Alli

### ISSUE THREAD

**Issue by se7en-illa** [hackforla/website#4314]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by roslynwythe** [comment]
Hi @se7en-illa I could not find you in the roster.  Please message me on Slack so I know which 'Sarah' is you, and I'll send you a link to complete the roster.  Thanks!

**Comment by se7en-illa** [comment]
**_How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?_**

It's taken me about a week and a few days to get up to my good first issue (finished with pull request created). I'm still waiting to attend my first meetings, since my onboarding was right before the beginning of the month (no meetings first 7 days). I will be in attendance for all meetings next week.

**Comment by se7en-illa** [comment]
Weekly Progress Update: 4/6/23
1. Progress: Finished pre-work up to making a PR for my first issue and reviewing my first PR.
2. Blockers: Waiting for PR for good first issue to be approved so I can continue on to my good second issue.
3. Availability: Monday - Friday, 12:00pm - 8:00pm EST (except meetings, which I am available to attend)
4. ETA: I should be able to complete pre-work by next Tuesday. (Need to attend first meetings).

**Comment by se7en-illa** [comment]
Weekly Progress Update: 4/14/23

Progress: Still need to attend team meetings.
Blockers: None for this week.
Availability: Monday - Friday, 12:00pm - 8:00pm EST (except meetings, which I am available to attend)
ETA: Pre-work will be complete next week.

**Comment by se7en-illa** [comment]
4/25/23 Pre-work completed!

**Comment by HackforLABot** [comment]
## Activity Log: se7en-illa
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- se7en-illa opened: https://github.com/hackforla/website/issues/4314 at 03/28/2023, 08:17 PM PDT
- se7en-illa assigned: https://github.com/hackforla/website/issues/4314 at 03/28/2023, 08:18 PM PDT
- se7en-illa assigned: https://github.com/hackforla/website/issues/4354 at 03/31/2023, 07:30 PM PDT
- se7en-illa unassigned: https://github.com/hackforla/website/issues/4354 at 03/31/2023, 07:30 PM PDT
- se7en-illa assigned: https://github.com/hackforla/website/issues/4395 at 04/05/2023, 09:43 AM PDT
- se7en-illa commented on: https://github.com/hackforla/website/issues/4395#issuecomment-1498437570 at 04/05/2023, 08:20 PM PDT
- se7en-illa opened: https://github.com/hackforla/website/pull/4430 at 04/06/2023, 09:46 AM PDT
- se7en-illa commented on: https://github.com/hackforla/website/issues/4395#issuecomment-1499349926 at 04/06/2023, 09:49 AM PDT
- se7en-illa commented on: https://github.com/hackforla/website/issues/4314#issuecomment-1499354277 at 04/06/2023, 09:54 AM PDT
- se7en-illa commented on: https://github.com/hackforla/website/issues/4314#issuecomment-1499361690 at 04/06/2023, 10:00 AM PDT
- se7en-illa commented on: https://github.com/hackforla/website/pull/4411#issuecomment-1499368671 at 04/06/2023, 10:06 AM PDT
- se7en-illa submitted review: https://github.com/hackforla/website/pull/4411#pullrequestreview-1375278124 at 04/06/2023, 10:14 AM PDT
- se7en-illa PR was merged: https://github.com/hackforla/website/pull/4430#event-8951306264 at 04/06/2023, 09:05 PM PDT
- se7en-illa issue closed by PR 4430: https://github.com/hackforla/website/issues/4395#event-8951306380 at 04/06/2023, 09:05 PM PDT
- se7en-illa assigned: https://github.com/hackforla/website/issues/4352 at 04/07/2023, 09:38 AM PDT
- se7en-illa commented on: https://github.com/hackforla/website/issues/4352#issuecomment-1500449393 at 04/07/2023, 09:40 AM PDT
- se7en-illa commented on: https://github.com/hackforla/website/issues/4314#issuecomment-1511746275 at 04/17/2023, 09:58 AM PDT
- se7en-illa opened: https://github.com/hackforla/website/pull/4539 at 04/17/2023, 10:14 AM PDT
- se7en-illa commented on: https://github.com/hackforla/website/issues/4352#issuecomment-1511773163 at 04/17/2023, 10:17 AM PDT
- se7en-illa commented on: https://github.com/hackforla/website/pull/4539#issuecomment-1513845679 at 04/18/2023, 03:04 PM PDT
- se7en-illa PR was merged: https://github.com/hackforla/website/pull/4539#event-9041883127 at 04/18/2023, 08:26 PM PDT
- se7en-illa issue closed by PR 4539: https://github.com/hackforla/website/issues/4352#event-9041883220 at 04/18/2023, 08:26 PM PDT
- se7en-illa assigned: https://github.com/hackforla/website/issues/4451 at 04/18/2023, 11:58 PM PDT
- se7en-illa commented on: https://github.com/hackforla/website/issues/4451#issuecomment-1514225930 at 04/18/2023, 11:59 PM PDT
- se7en-illa opened: https://github.com/hackforla/website/pull/4549 at 04/19/2023, 07:53 PM PDT
- se7en-illa PR was merged: https://github.com/hackforla/website/pull/4549#event-9060662153 at 04/20/2023, 12:06 PM PDT
- se7en-illa issue closed by PR 4549: https://github.com/hackforla/website/issues/4451#event-9060662345 at 04/20/2023, 12:06 PM PDT
- se7en-illa commented on: https://github.com/hackforla/website/pull/4558#issuecomment-1519119037 at 04/23/2023, 10:44 AM PDT
- se7en-illa commented on: https://github.com/hackforla/website/pull/4550#issuecomment-1519123489 at 04/23/2023, 11:07 AM PDT
- se7en-illa submitted review: https://github.com/hackforla/website/pull/4558#pullrequestreview-1401066940 at 04/25/2023, 07:38 PM PDT
- se7en-illa commented on: https://github.com/hackforla/website/issues/4314#issuecomment-1522691067 at 04/25/2023, 07:47 PM PDT
- se7en-illa closed as completed: https://github.com/hackforla/website/issues/4314#event-9098356951 at 04/25/2023, 07:47 PM PDT
- se7en-illa assigned: https://github.com/hackforla/website/issues/4551 at 04/25/2023, 07:51 PM PDT
- se7en-illa commented on: https://github.com/hackforla/website/issues/4551#issuecomment-1522694801 at 04/25/2023, 07:53 PM PDT
- se7en-illa commented on: https://github.com/hackforla/website/pull/4644#issuecomment-1544815110 at 05/11/2023, 04:17 PM PDT
- se7en-illa unassigned: https://github.com/hackforla/website/issues/4551 at 06/19/2023, 10:05 PM PDT
